src/blocks/plugins/css-handler/index.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { debounce } from 'lodash';
+
+import apiFetch from '@wordpress/api-fetch';
+
+import {
+	dispatch,
+	select,
+	subscribe
+} from '@wordpress/data';
+
+const { createNotice } = dispatch( 'core/notices' );
+
+const savePostMeta = debounce( async() => {
+	const { getCurrentPostId } = select( 'core/editor' );
+	const postId = getCurrentPostId();
+
+	createNotice(
+		'info',
+		__( 'Saving CSS…', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+
+	await apiFetch({ path: `themeisle-gutenberg-blocks/v1/save_post_meta/${ postId }`, method: 'POST' });
+
+	createNotice(
+		'info',
+		__( 'CSS saved.', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+}, 5000 );
+
+const saveWidgets = debounce( async() => {
+	createNotice(
+		'info',
+		__( 'Saving CSS…', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+
+	await apiFetch({ path: 'themeisle-gutenberg-blocks/v1/save_widgets_styles', method: 'POST' });
+
+	createNotice(
+		'info',
+		__( 'CSS saved.', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+}, 5000 );
+
+let reusableBlocks = {};
+
+subscribe( () => {
+	if ( select( 'core/edit-widgets' ) ) {
+		const {
+			isSavingWidgetAreas,
+			getEditedWidgetAreas
+		} = select( 'core/edit-widgets' );
+
+		const isSavingWidgets = isSavingWidgetAreas();
+		const editedAreas = getEditedWidgetAreas();
+
+		if ( isSavingWidgets && 0 < editedAreas.length ) {
+			saveWidgets();
+		}
+	}
+
+	if ( select( 'core/editor' ) ) {
+		const {
+			isCurrentPostPublished,
+			getEditedPostAttribute,
+			isSavingPost,
+			isPublishingPost,
+			isAutosavingPost,
+			__experimentalIsSavingReusableBlock
+		} = select( 'core/editor' );
+
+		const { __experimentalReusableBlocks } = select( 'core/block-editor' ).getSettings();
+
+		const { isSavingEntityRecord } = select( 'core' );
+
+		const { getBlocks } = select( 'core/block-editor' );
+
+		const { editPost } = dispatch( 'core/editor' );
+
+		const meta = getEditedPostAttribute( 'meta' ) || {};
+
+		if ( undefined !== meta._themeisle_gutenberg_block_has_review ) {
+			const hasReview = getBlocks().some( block => 'themeisle-blocks/review' === block.name );
+
+			if ( meta._themeisle_gutenberg_block_has_review !== hasReview ) {
+				editPost({
+					meta: {
+						'_themeisle_gutenberg_block_has_review': hasReview
+					}
+				});
+			}
+		}
```

Comment:
The property of the meta var is undefined. Thus the backend can not detect the posts that have the review.